src/main/webapp/index.html

```diff
@@ -260,5 +253,12 @@ <h4 class="titleStatus">Review&nbsp;
                 ?labelKeySearchTxt=xxxxxx
             </p>
         </div>
+        <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js">
+        </script>
+        <script type="text/javascript" src="js/bootstrap.js">
```

Comment:
It looks like bootstrap.js is deleted (rather than added) below -- is this correct or is the github diff just confusing me? No other concerns with this change.